fl4health/utils/losses.py

```diff
@@ -84,18 +204,20 @@ def clear(self) -> None:
         """
         raise NotImplementedError
 
-    @abstractmethod
     def compute(self) -> Losses:
         """
-        Compute aggregation of current list of losses if non-empty.
+        Compute average of current list of losses if non-empty.
```

Comment:
This isn't always the average if the meter type is just accumulation, which is why I think we used the term aggregation here originally.